src/microbe_stage/ProcessSystem.cs

```diff
@@ -128,7 +128,7 @@ public ProcessSystem(Node worldRoot)
         // Add movement consumption together
         result.BaseMovement = Constants.BASE_MOVEMENT_ATP_COST * hexCount;
         result.AddConsumption("baseMovement", result.BaseMovement);
-        var totalMovementConsumption = movementATPConsumption + result.BaseMovement;
+        result.TotalMovement = movementATPConsumption + result.BaseMovement;
```

Comment:
A bug you found while working on these changes I assume?

Reply:
I didn't look at the code, but I could have sworn that TotalMovement in the result *is* set correctly. This change would indicate that that variable is not initialized at all...

Reply:
Coming back a bit late, but indeed, it was never initialized at all... Which is why I had a few bugs at first.